### ISSUE TITLE
error logs in fabricator-assemble

### ISSUE THREAD

**Issue by lmolema** [fbrctr/fabricator#119]
I get an error when the assemble task executes, but I can't figure out where it is coming from. The error output is:

```
Error
    at Object.<anonymous> (/Users/USER/styleguide/node_modules/fabricator-assemble/node_modules/handlebars/dist/cjs/handlebars/exception.js:26:23)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
    at Module.load (module.js:355:32)
    at Function.Module._load (module.js:310:12)
    at Module.require (module.js:365:17)
    at require (module.js:384:17)
    at Object.<anonymous> (/Users/USER/styleguide/node_modules/fabricator-assemble/node_modules/handlebars/dist/cjs/handlebars/base.js:3:17)
    at Module._compile (module.js:460:26)
    at Object.Module._extensions..js (module.js:478:10)
```
I get this errors quite some times, but reversing my last changes did the trick. This time I can't get it working anymore. Any help? Is there a way that the error output is more specific about which file causes this error? Thanks in advance

**Comment by lmolema** [comment]
Ok, found it by commenting out each component and each structure and tracked the error eventually. Remains my question: Is there a way to see in the error log which file causes the assemble task to fail?

**Comment by LukeAskew** [comment]
There's definitely room for improvement in how [fabricator-assemble](https://github.com/fbrctr/fabricator-assemble) logs errors. Mind logging an issue over in that repo?

Out of curiosity, what was causing the error? Does it cause the gulp stream to break and the application to stop?

**Comment by lmolema** [comment]
Yes of course, I will log this issue in that repo, should done that in the first place :)

The gulp stream didn't break and the application was still available. Everything gets 'assembled' until the stream hits the error in the code. Everything afterwards does not 'assemble', but the application remains available.

The problem was I moved the checkbox.html component to the sub-folder form-elements, while it was included in a form.html structure later on. I had to change the path there to `{{> form-elements.checkbox}}`.
